design-guidelines/interaction-and-feedback.md

```diff
@@ -0,0 +1,59 @@
+# Interaction and Feedback
```

Comment:
"define UX guidelines for actions that cannot be executed"
What is title of this guideline?